internal/peeringdb/peeringdb.go

```diff
@@ -28,10 +28,23 @@ type Data struct {
 	ImportLimit6 int    `json:"info_prefixes6"`
 }
 
+var apiKey = ""
+
+func InitAPIKey(key string) {
+	if key != "" {
+		apiKey = key
+	}
+}
+
```

Comment:
This should be a parameter to `NetworkInfo` and `NeverViaRouteServers` instead of a global variable

Reply:
Fixed, Could you please check it again?

Thanks